CautionaryAlertsApi/V1/Gateways/UhGateway.cs

```diff
@@ -45,13 +47,51 @@ private List<PersonAlert> GetPersonAlerts(ContactLink link)
                 .Include(a => a.ContactLink)
                 .Where(a => a.ContactNumber == link.ContactNumber).ToList();
         }
-
         private CautionaryAlert GetDescriptionAndMapToDomain(PersonAlert alert)
         {
             var description = _uhContext.AlertDescriptionLookups
                 .OrderByDescending(a => a.DateModified)
                 .FirstOrDefault(a => a.AlertCode == alert.AlertCode);
             return alert.ToDomain(description?.Description);
         }
+
+
+        public CautionaryAlertsProperty GetCautionaryAlertsForAProperty(string propertyReference)
+        {
+            var addressLink = _uhContext.Addresses.
+                Where(x => x.PropertyReference == propertyReference).FirstOrDefault();
```

Comment:
Is there typically just one record in the database for each `propertyReference`? I'm wondering if we want to apply some sort of ordering before we take the first record.

Reply:
I believe it's always one record, I will check the data again.

Reply:
Only less than 10 records have a duplicate prop ref and none of them relating to a cautionary alert so for now I think we can stick with first or default.